.rubocop.yml

```diff
@@ -155,6 +155,11 @@ RSpec/EmptyExampleGroup:
 RSpec/ExampleLength:
   Enabled: false
 
+RSpec/FilePath:
+  Exclude:
+    - "spec/organizers/api_entreprise/user/oauth_login_spec.rb"
+    - "spec/organizers/api_particulier/user/oauth_login_spec.rb"
+
```

Comment:
J'aurais rename les fichiers ici

Reply:
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecfilepath

This cop is deprecated. We plan to remove it in the next major version update to 3.0. The migration targets are RSpec/SpecFilePathSuffix and RSpec/SpecFilePathFormat. If you are using this cop, please plan for migration.

Reply:
suppr dans le dernier commit